plugins_setup.py

```diff
@@ -30,43 +32,21 @@
 # under which the OpenSSL Project distributes the OpenSSL toolkit software,
 # as those licenses appear in the file LICENSE-OPENSSL.
 
-from metaswitch.clearwater.config_manager.plugin_base import ConfigPluginBase, FileStatus
-from metaswitch.clearwater.etcd_shared.plugin_utils import run_command, safely_write
-from time import sleep
 import logging
-import shutil
-import os
-
-_log = logging.getLogger("shared_config_plugin")
-_file = "/etc/clearwater/shared_config"
-
-class SharedConfigPlugin(ConfigPluginBase):
-    def __init__(self, _params):
-        pass
-
-    def key(self):
-        return "shared_config"
-
-    def file(self):
-        return _file
-
-    def status(self, value):
-        try:
-            with open(_file, "r") as ifile:
-                current = ifile.read()
-                if current == value:
-                    return FileStatus.UP_TO_DATE
-                else:
-                    return FileStatus.OUT_OF_SYNC
-        except IOError:
-            return FileStatus.MISSING
-
-    def on_config_changed(self, value, alarm):
-        _log.info("Updating shared configuration file")
-
-        if self.status(value) != FileStatus.UP_TO_DATE:
-            safely_write(_file, value)
-            run_command("/usr/share/clearwater/clearwater-queue-manager/scripts/modify_nodes_in_queue add apply_config")
-
-def load_as_plugin(params):
-    return SharedConfigPlugin(params)
+import sys
+import multiprocessing
+
+from setuptools import setup, find_packages
+
+setup(
+    name='clearwater-etcd-plugin-tests',
+    version='1.0',
+    namespace_packages = ['metaswitch'],
+    packages=['metaswitch', 'metaswitch.clearwater', 'metaswitch.clearwater.plugin_tests','clearwater_etcd_plugins','clearwater_etcd_plugins.chronos'],
```

Comment:
I don't see any reason for why it wouldn't. Should just be a case of adding `clearwater_etcd_plugins.<component>` to the packages, and a test file in plugin_tests.
Am i missing anything?